### ISSUE TITLE
look at ways we can persist the GPX files that we import

### ISSUE THREAD

**Issue by mcplanner** [digidem/ecuador-map-editor#9]

**Comment by gmaclennan** [comment]
I don't think this is needed right now. The following workflow works with current GPS data:

1. Copy GPX files off GPS and organize in folders by GPS. The GPS automatically names the latest waypoints in a new file every time it is connected to the computer.
2. Drag the files one-by-one onto iD to add a point, line or area and go through the notebook adding type and metadata for each point.
3. Manually manage the GPX files to keep track of which ones have been processed and entered.

I think longer term we should see GPX points the same as observations, and track them in the database the same way. They should ideally be linked with the places we create in iD editor, so that we can always go back to the original GPS point that was used to create the place.
